src/platforms/hosted/remote_jtagtap.c

```diff
@@ -156,7 +149,7 @@ static void jtagtap_tdi_tdo_seq(
 		}
 		if (DO) {
 			uint64_t res = remotehston(-1, (char *)&construct[1]);
-			memcpy(DO, &res, bytes);
+			memcpy(DO, &res, bytes); /* Fixme: Endian dependant!*/
```

Comment:
the memcpy is right - you can use https://github.com/bad-alloc-heavy-industries/substrate/blob/7fa29f823130c40ea0b10f393537b680c6d4f81e/impl/socket.cxx#L133 as a basis for how to fix this (yes, this is C++, but the fixed stack array and platform-independent population of it still applies here). The compiler is very good at reading through the code and generating bswap/xchng operations as required

Reply:
Committed not the latest version...